### PR TITLE
Roll up typescript with Go

### PR DESCRIPTION
**Pull request by chaodaiG**
Following #25556 , another attempt to stabilize ts rollup speed

- easier to maintain than bash
- can do things in parallel

only use it for deck at this point, will use ii for other things if apply

/cc @cjwagner @BenTheElder

### CODE REVIEW

**Review by cjwagner — APPROVED** [review]
/hold

**Comment by chaodaiG** [comment]
the build duration is not too bad right now, retest to ensure this does help
/test all

**Comment by chaodaiG** [comment]
reducing image building parallelism from 5 to 10 actually slowed things down, so restored back to 10

File: hack/run-in-node-container.sh [hunk 1]
```diff
@@ -41,6 +41,7 @@ fi
 # We are running as the current host user/group so the files produced are
 # owned appropriately on the host.
 "${DOCKER[@]}" run \
+    --cpus=0.000 \
```
Comment (BenTheElder) [inline hack/run-in-node-container.sh:44]: 🤨

Comment (chaodaiG) [reply hack/run-in-node-container.sh:44]: A line that I desperately added, and turned out that the total root cause of flakiness was using highcpu-32 nodes, I'll revert this change 🙃

Comment (chaodaiG) [reply hack/run-in-node-container.sh]: removed

**Review by cjwagner — APPROVED** [review]

**Comment by BenTheElder** [comment]
@chaodaiG it still has a /hold so it's not merging (guessing at /test all motivation)

**Comment by chaodaiG** [comment]
> @chaodaiG it still has a /hold so it's not merging (guessing at /test all motivation)

Just to make sure that this change isn't gonna make presubmit tests flaky or slower
/unhold

**Pull request merged**